maven-core/src/main/java/org/apache/maven/project/collector/MultiModuleCollectionStrategy.java

```diff
@@ -76,7 +76,7 @@ public MultiModuleCollectionStrategy( ModelLocator modelLocator, ProjectsSelecto
             }
             else
             {
-                logger.debug( "Multi module project collection failed: {}"
+                LOGGER.debug( "Multi module project collection failed:{}"
```

Comment:
Please restore the space after the colon.

Reply:
Have a look at the first arg: line separator.